baybe/utils/chemistry.py

```diff
@@ -149,10 +149,14 @@ def smiles_to_fingerprint_features(
     )
     name = f"{encoding.name}_"
     prefix = prefix + "_" if prefix else ""
-    col_names = [
-        prefix + name + f.split("fingerprint")[1]
+    no_descriptor_names = all(
+        "fingerprint" in f for f in fingerprint_encoder.get_feature_names_out()
+    )
+    suffixes = [
+        f.split("fingerprint")[1] if no_descriptor_names else f
```

Comment:
can you remind me: for which encoding do we actually end up in the second case of the if-condition?

Reply:
![image](https://github.com/user-attachments/assets/034bf418-7713-4899-b046-9902ebb461ed)

Reply:
I guess that we still stick with the current implementation then? That is, not have anything on this hard-coded? Can this the be resolved?